workbox-config.js

```diff
@@ -19,7 +19,7 @@
     },
   },
   {
-    urlPattern: new RegExp('^https://gateway.marvel.com/v1/public/.+'),
+    urlPattern: new RegExp(/https\:\/\/gateway\.marvel\.com\/v1\/public\/.+/g),
```

Comment:
**Suggestion:** The RegExp constructor is not necessary when defining a regular expression. It can be simplified.
```suggestion
    urlPattern: /https\:\/\/gateway\.marvel\.com\/v1\/public\/.+/g,
```